README.md

```diff
@@ -588,6 +588,9 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [3D-Icons](https://free3dicon.com/): Free 3D - Icons to download
 - [533 Hand-Picked Tools & Resources for Web Designers & Developers](https://toolkit.addy.codes/)
 - [Illlustrations](https://illlustrations.co/): Awesome 120+ free illustrations
+- [remove.bg](https://www.remove.bg/): Remove background from images and videos quickly
```

Comment:
There are tools that are provide better features for free, so please remove this one